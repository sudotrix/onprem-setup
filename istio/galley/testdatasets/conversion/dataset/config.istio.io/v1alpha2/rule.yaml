apiVersion: "config.istio.io/v1alpha2"
kind: rule
metadata:
  name: valid-rule
spec:
  actions:
  - handler: my-handler
    instances: [ my-instance ]
