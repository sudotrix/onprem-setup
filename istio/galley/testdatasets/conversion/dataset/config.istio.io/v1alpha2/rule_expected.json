{
  "istio/policy/v1beta1/rules": [
    {
      "Metadata": {
        "name": "{{.Namespace}}/valid-rule"
      },
      "Body": {
        "actions": [
          {
            "handler": "my-handler",
            "instances": [
              "my-instance"
            ]
          }
        ]
      },
      "TypeURL": "type.googleapis.com/istio.policy.v1beta1.Rule"
    }
  ]
}
